src/renderRepoCard.js

```diff
@@ -62,28 +92,12 @@ const renderRepoCard = (repo, options = {}) => {
       <text x="50" y="38" class="header">${header}</text>
       <text class="description" x="25" y="70">${encodeHTML(desc)}</text>
       
-      <g transform="translate(30, 100)">
-        <circle data-testid="lang-color" cx="0" cy="-5" r="6" fill="${langColor}" />
-        <text data-testid="lang" class="gray" x="15">${langName}</text>
+      ${svgLanguage}
+      
+      <g transform="translate(${155 - shiftText}, 100)">
+        ${FlexLayout({ items: [svgStars, svgForks], gap: 65 }).join("")}
       </g>
```

Comment:
I'm not putting `svgLanguage` into the FlexLayout's item array because it breaks the layout since i cannot know the svgLanguage's width before hand. I know i can calculate the text length but it does not solves the issue entirely since the FlexLayout then would relay on texts instead of actual element's dimensions